smarty/templates/install.tpl

```diff
@@ -3,8 +3,8 @@
 	{* installdb.php is in the root directory, so we know these relative
 	   links should work. Load in the base LORIS bootstrap CSS
 	*}
-		<link rel="stylesheet" href="bootstrap/css/bootstrap.min.css">
-		<link rel="stylesheet" href="bootstrap/css/custom-css.css">
+		<link rel="stylesheet" href="/bootstrap/css/bootstrap.min.css">
+		<link rel="stylesheet" href="/bootstrap/css/custom-css.css">
```

Comment:
What's the reason for this change? It's going to make the installer break if LORIS isn't hosted on the root of the web server. (It also makes the comment above it confusing..)

Reply:
The page wasn't loading css otherwise. I will look at using baseurl if possible